### ISSUE TITLE
Enhance the style of scrollbars

### ISSUE THREAD

**Issue by miina** [GoogleForCreators/web-stories-wp#3363]
## Feature Description

This is a follow-up issue for https://github.com/google/web-stories-wp/issues/1698 which was implemented in https://github.com/google/web-stories-wp/pull/1858, however, since the UX of the default OS scrollbars is not ideal in the editor context, the scrollbars need a style enhancement instead.

From Paul:
> The downside, as is correctly pointed out, is that the auto-hiding scrollbar is the OS level scrollbar which cannot be styled. Thus, if @samitron7's requirement is proper styling for the scrollbar no matter what, we have two choices:
> 1. Compromise on styling, but do auto-hide (note: the always-visible scrollbar would still be styled)
> 2. Style the scrollbars, but make them thin and always visible (roughly what we already have)

The decision was to go with 2, see the full discussion thread [starting from here](https://github.com/google/web-stories-wp/pull/1858#discussion_r456186463).

## Alternatives Considered

- Auto-hiding scrollbars which was discarded due to styling.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by o-fernandez** [comment]
@samitron7 is this important for the stable launch? Or can it be done post the initial release? Also, is it already spec'd in the visual design work (and can be closed?) or does it still require a separate ticket? 

Also @barklund in case you know.
